### ISSUE TITLE
fail: CI is failing to publish docs

### ISSUE THREAD

**Issue by octonato** [akka/akka-http#4371]
Being caused by a `NoSuchElementException` from an Option when generated javadoc. 
Probably being triggered by some recent dependency update.

https://github.com/akka/akka-http/actions/runs/8464795725/job/23190171387#step:4:506

```scala
[error] java.util.NoSuchElementException: None.get
[error] 	at scala.None$.get(Option.scala:627)
[error] 	at scala.None$.get(Option.scala:626)
[error] 	at com.typesafe.genjavadoc.BasicTransform.deprecationInfo(BasicTransform.scala:163)
[error] 	at com.typesafe.genjavadoc.BasicTransform.deprecationInfo(BasicTransform.scala:159)
[error] 	at com.typesafe.genjavadoc.BasicTransform.$anonfun$addMethod$1(BasicTransform.scala:152)
[error] 	at scala.Option.map(Option.scala:242)
[error] 	at com.typesafe.genjavadoc.BasicTransform.addMethod(BasicTransform.scala:152)
[error] 	at com.typesafe.genjavadoc.BasicTransform.newTransform(BasicTransform.scala:111)
[error] 	at com.typesafe.genjavadoc.BasicTransform.newTransform$(BasicTransform.scala:49)
[error] 	at com.typesafe.genjavadoc.GenJavadocPlugin$MyComponent$GenJavadocTransformer.newTransform(Plugin.scala:93)
[error] 	at com.typesafe.genjavadoc.GenJavadocPlugin$MyComponent$GenJavadocTransformer.transform(Plugin.scala:103)
[error] 	at com.typesafe.genjavadoc.GenJavadocPlugin$MyComponent$GenJavadocTransformer.transform(Plugin.scala:93)
[error] 	at scala.reflect.api.Trees$Transformer.$anonfun$transformStats$1(Trees.scala:2625)
[error] 	at scala.reflect.api.Trees$Transformer.transformStats(Trees.scala:2623)
[error] 	at scala.reflect.internal.Trees$Template.transform(Trees.scala:546)
[error] 	at scala.reflect.internal.Trees$InternalTransformer.transform(Trees.scala:1489)
[error] 	at com.typesafe.genjavadoc.GenJavadocPlugin$MyComponent$GenJavadocTransformer.superTransform(Plugin.scala:102)
[error] 	at com.typesafe.genjavadoc.BasicTransform.newTransform(BasicTransform.scala:116)
[error] 	at com.typesafe.genjavadoc.BasicTransform.newTransform$(BasicTransform.scala:49)
```

**Comment by octonato** [comment]
Well, very recent.
https://github.com/akka/akka-http/pull/4369

Last merged PR.

**Comment by octonato** [comment]
Oh, sorry for the noise. The PR is the fix for the issue. And I was looking at old CI reports. Must be fixed by #4369. 

I will close this one and get some more coffee to wake up.
